otx/cli/tools/train.py

```diff
@@ -144,15 +135,50 @@ def parse_args():
         help="Total number of workers in a worker group.",
     )
 
-    add_hyper_parameters_sub_parser(parser, hyper_parameters)
-
-    return parser.parse_args(), template, hyper_parameters
+    return parser
 
 
-def main():  # pylint: disable=too-many-branches
+# pylint: disable=too-many-branches, too-many-statements
+def main():
     """Main function that is used for model training."""
-    # Dynamically create an argument parser based on override parameters.
-    args, template, hyper_parameters = parse_args()
+    parser = get_parser()
+
+    args, _ = parser.parse_known_args()
+    train_workspace_path = os.getcwd()
+
+    has_template_yaml = Path(args.template).exists()
+    has_data_yaml = Path(args.data).exists()
+    was_built_before = has_template_yaml and has_data_yaml
```

Comment:
how about `use_workspace = Path(args.template).exists() and Path(args.data).exists()`?
and `args.data` and `args.template` are always non-None, right?

Reply:
In the below line, we need to check whether there is `template.yaml` or not. So, I implemented this way.
The default value of `args.data` and `args.template` is `./data.yaml` and `./template.yaml`, respectively.

Reply:
Yes, I understand. How about using `use_workspace` instead of `was_built_before`?

Reply:
For the name of variable, Sure :)